src/pages/admin/main/components/title-block/TitleBlockForm.module.scss

```
@@ -0,0 +1,56 @@
+    font-size: 0.875rem;
+    margin: 0;
+}
+
+.text-section {
+    flex: 4;
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+}
+
+.textarea-custom :global(.textarea) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)